assets/constants.ts

```diff
@@ -5585,7 +5585,7 @@ export default {
   "MAX_CHARS_IN_SKILL_DESCRIPTION": 100,
   "MAX_CHARS_IN_STORY_TITLE": 39,
   "MAX_CHARS_IN_STORY_DESCRIPTION": 1000,
-  "MAX_CHARS_IN_CHAPTER_TITLE": 36,
+  "MAX_CHARS_IN_LESSON_TITLE": 36,
```

Comment:
Maybe better to call it MAX_CHARS_IN_EXPLORATION_TITLE, to avoid confusion of what a "lesson" is?